input/pagecontent/vrdr-content-transitions.md

```diff
@@ -2,56 +2,76 @@
 
 
 
-| Name of Profile | Group | change  |  Current Profile   | New Profile  | Comment |
-| :-------------: | ----- | ------- | ------------------ | ------------ | :-----: | 
+| Name of Profile  | Group | change  |  Current Profile   | New Profile  | Comment |
+| ------------- | ----- | ------- | ------------------ | ------------ | ----- | 
 |Death Certificate |   -  |    -    | [DeathCertificate] |     -         |   -      |
 |Decedent |   DecedentDemographics  |    -    | [Decedent] |       -       |    -     |
 |Father |     DecedentDemographics    |    -    | [DecedentFather] |       -       |    -     |
 |Mother |     DecedentDemographics    |    -    | [DecedentMother] |       -       |    -     |
 |Spouse |     DecedentDemographics    |    -    | [DecedentSpouse] |       -       |    -     |
 |Age at Death |     DecedentDemographics    |    -    | [DecedentAge] |       -       |    -     |
 |BirthRecord ID |     DecedentDemographics    |    -    | [BirthRecordIdentifier] |       -       |    -     |
-|Education Level |     DecedentDemographics    |    -    | [DecedentEducationLevel] |       -       |    -     |
+|Education Level |     DecedentDemographics    |    -    | [DecedentEducationLevel] |       [ObservationEducationLevelVitalRecordsNew]      |    -     |
 |Military Service |    DecedentDemographics    |    -    | [DecedentMilitaryService] |       -       |    -     |
 |Usual Work |     DecedentDemographics   |    -    | [DecedentUsualWork] |       -       |    -     |
 | Emerging Issues |     DecedentDemographics   |    -    | [EmergingIssues] |       -       |    -     |
-| Input Race and Ethnicity |     DecedentDemographics    |    -    | [InputRaceAndEthnicity] |       -       |    -     |
+| Input Race and Ethnicity |     DecedentDemographics    |    -    | [InputRaceAndEthnicity] |       [InputRaceAndEthnicityNew]       |    -     |
 | Examiner Contacted |     DeathInvestigation    |    -    | [ExaminerContacted] |       -       |    -     |
-|Pregnancy Status |     DeathInvestigation    |    -    | [DecedentPregnancyStatus] |       -       |    -     |
-|Tobacco Use |     DeathInvestigation    |    -    | [TobaccoUseContributedToDeath] |       -       |    -     |
+|Pregnancy Status |     DeathInvestigation    |    -    | [DecedentPregnancyStatus] |       [ObservationDecedentPregnancyVitalRecordsNew]       |    -     |
+|Tobacco Use |     DeathInvestigation    |    -    | [TobaccoUseContributedToDeath] |       [ObservationTobaccoUseContributedToDeathVitalRecords]       |    -     |
 |Military Service |     DeathInvestigation    |    -    | [DeathCertificate] |       -       |    -     |
```

Comment:
military service is here twice (shouldn't be under DeathInvestigation)

also dispositionLocation seems to be missing (under DecedentDisposition) 

PlaceOfInjury is located under coded content, but is also a slice under DeathInvestigation. Not sure if we need to have both

Reply:
Good catches...